packages/types/src/index.ts

```diff
@@ -267,6 +267,14 @@ export interface Timepicker extends Action {
   timezone?: string;
 }
 
+export interface DateTimepicker extends Action {
+  type: 'datetimepicker';
+  initial_date_time?: number;
+  confirm?: Confirm;
+  focus_on_load?: boolean;
+  timezone?: string;
```

Comment:
Why did you add this property? It's not available at least in the public document.

Reply:
Ah my bad, copy paste leftover 🙈